tests/foreman/api/test_registration.py

```diff
@@ -227,3 +227,26 @@ def test_negative_global_registration_without_ak(module_target_sat):
     with pytest.raises(HTTPError) as context:
         module_target_sat.api.RegistrationCommand().create()
     assert 'Missing activation key!' in context.value.response.text
+
+
+def test_positive_capsule_without_registration_module(module_target_sat, module_capsule_configured):
+    """Test a capsule without registration module
+
+    :id: a2f23e42-648d-4428-a961-6e0b933c6dff
+
+    :steps:
+
+            1. Get a configured capsule
+            2. Set the registration to false for the capsule
+            3. Verify the result
```

Comment:
I think intent of this test should be to verify if registration is possible when registration is disabled on capsule, right?

Reply:
I updated the test to add the registration.